### PR TITLE
Feature/129032 add type of concern

### PR DESCRIPTION
**Pull request by nimble-dfe**
**What is the change?**
Non-concerns journey
**Why do we need the change?**
Ability to  add a concern to a non concerns case
**What is the impact?**
Page change
**Azure DevOps Ticket**
https://dfe-gov-uk.visualstudio.com/Academies-and-Free-Schools-SIP/_boards/board/t/Concerns%20Casework/Stories/?workitem=129032
https://dfe-gov-uk.visualstudio.com/Academies-and-Free-Schools-SIP/_boards/board/t/Concerns%20Casework/Stories/?workitem=129033
https://dfe-gov-uk.visualstudio.com/Academies-and-Free-Schools-SIP/_boards/board/t/Concerns%20Casework/Stories/?workitem=129034

### CODE REVIEW

File: ConcernsCaseWork/ConcernsCaseWork.Data/Gateways/ConcernsCaseGateway.cs [hunk 1]
```diff
@@ -14,8 +14,17 @@ public ConcernsCaseGateway(ConcernsDbContext concernsDbContext)
 
         public ConcernsCase SaveConcernsCase(ConcernsCase concernsCase)
         {
-            _concernsDbContext.ConcernsCase.Update(concernsCase);
-            _concernsDbContext.SaveChanges();
+	        try
```
Comment (mikestock-nimble) [inline ConcernsCaseWork/ConcernsCaseWork.Data/Gateways/ConcernsCaseGateway.cs:17]: QUESTION:

Do we need the try catch if we just re-throw the exception?

File: ConcernsCaseWork/ConcernsCaseWork.Tests/Pages/Case/Concern/IndexPageModelTests.cs [hunk 2]
```diff
@@ -1,426 +1,426 @@
-﻿using ConcernsCaseWork.Authorization;
```
Comment (mikestock-nimble) [inline ConcernsCaseWork/ConcernsCaseWork.Tests/Pages/Case/Concern/IndexPageModelTests.cs:1]: QUERY:

Is this file meant to be commented?

File: ConcernsCaseWork/ConcernsCaseWork.Tests/Pages/Case/Management/Concern/IndexPageModelTests.cs [hunk 3]
```diff
@@ -73,7 +73,7 @@ public async Task WhenOnGetAsync_ReturnPage()
 			Assert.That(pageModel.CreateRecordsModel.Count, Is.EqualTo(createRecordsModel.Count));
```
Comment (mikestock-nimble) [inline ConcernsCaseWork/ConcernsCaseWork.Tests/Pages/Case/Management/Concern/IndexPageModelTests.cs:73]: QUERY:

Are these test assertions mean to be commented?

File: ConcernsCaseWork/ConcernsCaseWork.Tests/Pages/Case/Management/IndexPageModelTests.cs [hunk 4]
```diff
@@ -81,64 +84,7 @@ public async Task WhenOnGetAsync_MissingCaseUrn_ThrowsException_ReturnPage()
 				c.GetTrustByUkPrn(It.IsAny<string>()), Times.Never);
 		}
 
-		[Test]
-		public async Task WhenOnGetAsync_ModelDataIsBuilt()
```
Comment (mikestock-nimble) [inline ConcernsCaseWork/ConcernsCaseWork.Tests/Pages/Case/Management/IndexPageModelTests.cs:85]: QUERY:

Should this test have been deleted?

**Review by mikestock-nimble — CHANGES_REQUESTED** [review]

File: ConcernsCaseWork/ConcernsCaseWork.Data/Gateways/ConcernsCaseGateway.cs (hunk 1 above)
Comment (nimble-dfe) [reply ConcernsCaseWork/ConcernsCaseWork.Data/Gateways/ConcernsCaseGateway.cs:17]: debug code, removed

**Pull request merged**